src/texts.js

```diff
@@ -77,3 +50,32 @@ export function getText(lang, name, placeholders = {}) {
     text,
   )
 }
+
+let DATA = null;
```

Comment:
I'm kinda ok with this name as it is only used in this file, but I think it could be better. Something like "LOADED_TEXTS"?
Also I thought about the initializing of this, but couldn't really find a better idea.

Reply:
Actually, there was const declared before with the same name :scream:. I renamed the variable to TEXT_MESSAGES

Reply:
Actually, there was const declared before with the same name :scream:. I renamed the variable to TEXT_MESSAGES